packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -102,6 +104,7 @@ export const Autocomplete = <T extends {}>({
   onChange,
   onQueryChange,
   placeholder = 'Search',
+  selectedItem,
```

Comment:
```suggestion
  selectedItem = '',
```
Since you set it as optional on the interface